hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/FieldOperations.java

```diff
@@ -31,831 +31,829 @@
 import java.util.function.BiConsumer;
 
 /**
- * Should always be consistent with {@link com.hazelcast.nio.serialization.FieldType}
+ * Should always be consistent with {@link FieldKind}
  */
-@SuppressWarnings("checkstyle:AnonInnerLength")
-public enum FieldOperations {
+@SuppressWarnings({"checkstyle:AnonInnerLength", "checkstyle:ExecutableStatementCount"})
+public final class FieldOperations {
```

Comment:
I personally find the old enum approach better for readability. Is there a reason we changed it to this?

Reply:
I just though this is more straightforward. Not a strong reason.